consensus/types/src/beacon_block_header.rs

```diff
@@ -13,6 +13,7 @@ use tree_hash_derive::TreeHash;
 #[derive(
     arbitrary::Arbitrary,
     Debug,
+    Default,
```

Comment:
This doesn't seem to be used

Reply:
removed in https://github.com/sigp/lighthouse/pull/4946/commits/d314d12a0f23b65df1d953e18778fdf59d327bfb